Rakefile

```diff
@@ -1,7 +1,7 @@
 require 'open-uri'
 require 'bundler/gem_tasks'
 
-UPSTREAM_VERSION = '0.1.3'
```

Comment:
You'll also need to change lines 25 - 30 to get this to work. 
The error is most likely linked to the M1 processors on Mac than the OS.

Reply:
Thanks for the tip, @frankywahl I'll check it out. I experienced this error however on an Intel chip. We've had a few folks suspecting the M1 though so I can have them run through this on their machines and see what we get.

Nice to see you here btw. Hope things are well 😄

Reply:
I haven't personally tested M1's yet, but I think what I have will work. If it doesn't I'll submit a follow up release to fix it.

@SP-UT since you have an M1, mind giving this a try?